zerver/lib/email_notifications.py

```diff
@@ -707,31 +708,58 @@ def get_onboarding_email_schedule(user: UserProfile) -> Dict[str, timedelta]:
     # -Do not send emails on Saturday or Sunday
     # -Have at least one weekday between each (potential) email
 
+    # User signed up on Monday
+    if signup_day == 1:
+        # Send onboarding_zulip_topics on Wednesday
+        # Send onboarding_zulip_guide on Friday
+        # Send onboarding_team_to_zulip on Tuesday
+        onboarding_emails["onboarding_team_to_zulip"] = timedelta(days=8, hours=-1)
+
     # User signed up on Tuesday
     if signup_day == 2:
         # Send onboarding_zulip_topics on Thursday
         # Send onboarding_zulip_guide on Monday
         onboarding_emails["onboarding_zulip_guide"] = timedelta(days=6, hours=-1)
+        # Send onboarding_team_to_zulip on Wednesday
+        onboarding_emails["onboarding_team_to_zulip"] = timedelta(days=8, hours=-1)
 
     # User signed up on Wednesday
     if signup_day == 3:
         # Send onboarding_zulip_topics on Friday
         # Send onboarding_zulip_guide on Tuesday
         onboarding_emails["onboarding_zulip_guide"] = timedelta(days=6, hours=-1)
+        # Send onboarding_team_to_zulip on Thursday
+        onboarding_emails["onboarding_team_to_zulip"] = timedelta(days=8, hours=-1)
 
     # User signed up on Thursday
     if signup_day == 4:
         # Send onboarding_zulip_topics on Monday
         onboarding_emails["onboarding_zulip_topics"] = timedelta(days=4, hours=-1)
         # Send onboarding_zulip_guide on Wednesday
         onboarding_emails["onboarding_zulip_guide"] = timedelta(days=6, hours=-1)
+        # Send onboarding_team_to_zulip on Friday
+        onboarding_emails["onboarding_team_to_zulip"] = timedelta(days=8, hours=-1)
 
     # User signed up on Friday
     if signup_day == 5:
         # Send onboarding_zulip_topics on Tuesday
         onboarding_emails["onboarding_zulip_topics"] = timedelta(days=4, hours=-1)
         # Send onboarding_zulip_guide on Thursday
         onboarding_emails["onboarding_zulip_guide"] = timedelta(days=6, hours=-1)
+        # Send onboarding_team_to_zulip on Monday
+        onboarding_emails["onboarding_team_to_zulip"] = timedelta(days=10, hours=-1)
+
+    # User signed up on Saturday
+    # Send onboarding_zulip_topics on Monday
+    # Send onboarding_zulip_guide on Wednesday
+    # Send onboarding_team_to_zulip on Friday
```

Comment:
Is this unfinished?

Reply:
It was a code comment I used when I was double checking the email schedule logic. I've updated it to just say no adjustments are needed if the user signs up on Saturday.